src/main/resources/add-ons-to-index.json

```diff
@@ -43,6 +43,27 @@
       }
     },
     {
+    "uid": "org.openmrs.owa.addonmanager",
+    "type": "OWA",
+    "name": "Add On Manager OWA",
+    "maintainers": [
+      {
+        "name": "Andela apprentices",
+        "url": "https://andela.com/"
+      },
+      {
+        "name": "OpenMRS",
+        "url": "https://github.com/openmrs"
+      }
+    ],
+    "backend": "org.openmrs.addonindex.backend.Bintray",
+    "bintrayPackageDetails": {
+      "owner": "openmrs",
+      "repo": "owa",
+      "package": "openmrs-addOnManager"
```

Comment:
Have you looked at the formate of the package for other OWAs?

Reply:
Its fixed

Reply:
Are you done with the other ticket of changing the addon display name and download zip file?

Reply:
Yes am done.